### ISSUE TITLE
How to navigate through the help window that emerges in auto-completion mode

### ISSUE THREAD

**Issue by jrzaurin** [emacs-ess/ESS#139]
Hi there, 

I have been using ESS on a mac 10.9.5 (emacs 24.4), and I quite like it. Is like "all in one place", making my life easier. I just have one question regarding on how to access to the help window that emerges after activating auto-completion.  Here I attach a screen capture. I refer to the yellow window in the image.  
![screen shot 2015-01-26 at 11 29 43](https://cloud.githubusercontent.com/assets/7262383/5899093/d18bf0d2-a54e-11e4-931f-e14d219f929a.png)

I find it quite useful, but I can't navigate through that window so I can see all the help file (right now it only goes down to "Arguments" for the particular case shown here). I have tried different keys and key-combinations, but every time I type something or I scroll on it, disappears.   

Thanks in advance for your time!

**Comment by vforget** [comment]
I am by no means an ESS/emacs expert, but from searching around appears
that the tooltip is not scrollable. Also note lack of scroll bar, where as
popop menu does have scroll bar when length of items > 10. Try by
completing "plot.".

Also, see auto-complete manual, and you will notice no option to scroll the
tooltip (called quick-help).

Also from manual see:

http://auto-complete.org/doc/manual.html#ac-quick-help-height

Which can modify the height of the tooltip, so you can see more
documentation:

(setq ac-quick-help-height 50)

**Comment by vspinu** [comment]
Yes. I think there is no technical way to scroll through that window. I agree that it would be useful but given that auto-complete is gradually fading in favor of company-mode I don't think it's worth investing too much time on this.

**Comment by jrzaurin** [comment]
Hi guys, many thanks, 

@vforget : I tried the (setq ac-quick-help-height 50) and certainly improves. Thanks for all your insight. I actually looked around, but i did not find much information, so I assumed that was super trivial and that I was missing something. 

@vspinu : I am not giving a go to company and it looks pretty similar to auto-complete. But if company is the future, let's go for it!

Thanks again!
